docs/UserGuide.md

```diff
@@ -89,21 +105,18 @@ If you can type fast, TA<sup>2</sup> can get your contact and assignment managem
 
 ### Viewing help: `help`
 
-Shows a help window with a link directing you to the user guide. Alternatively this can be access by clicking the `Help` button
-that can be found in the top left corner.
+Shows a help window with a link directing you to the user guide. Alternatively, you can access this page by clicking the `Help` button in the top left corner.
```

Comment:
clicking Help F1 after clicking Help in top left corner, but could we make Help F1 stuff gone to achieve simply clicking Help

Reply:
how about include both?